packages/client/README.md

```diff
@@ -114,257 +205,280 @@ const msg = {
 }
 
 // Publish using the stream id only
-await client.publish('my-stream-id', msg)
+await streamr.publish(streamId, msg)
 
 // The first argument can also be the stream object
-await client.publish(stream, msg)
+await streamr.publish(stream, msg)
 
 // Publish with a specific timestamp as a Date object (default is now)
-await client.publish('my-stream-id', msg, new Date(54365472))
+await streamr.publish(streamId, msg, new Date(54365472))
```

Comment:
Are these timestamps realistic data? Could use the timestamp of the other example